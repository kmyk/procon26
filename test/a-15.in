01111111111111111111111111111110
11111111111111111111111111111111
11011111111111111111111111111111
11011111111111111111111111111111
11000111111111111111111111111111
11010000001111111111111111111111
11010111101111111111111111111111
11111111101111111111111111111111
11111111101111111111111111111111
11111111101111111111111111111111
11111111100000011111111111111111
11111111111111011111111111111111
11111111111111011111111111111111
11111111111111011111111111111111
11111111111111011111111111111111
11111111111111000000111111111111
11111111111111111110111111111111
11111111111111111110111111111111
11111111111111111110111111111111
11111111111111111110111111111111
11111111111111111110000001111111
11111111111111111111111101111111
11111111111111111111111101111111
11111111111111111111111101111111
11111111111111111111111101111111
11111111111111111111111101111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
01111111111111111111111111111110

9
00000000
00000000
00000000
00000100
00000100
00000111
00000101
00000101

00000000
00000000
00000000
00000100
00000100
00000100
00000100
00000100

00000001
00000001
00000001
00000001
00000001
00000000
00000000
00000000

00000000
00000000
00000000
00000000
00000000
00000000
00000000
00011111

11111000
00000000
00000000
00000000
00000000
00000000
00000000
00000000

10000000
10000000
10000000
10000000
10000000
00000000
00000000
00000000

00000000
00011111
00000000
00000000
00000000
00000000
00000000
00000000

00000000
00000000
00000000
00000000
00000000
00000000
00000000
11111000

00000000
00000000
00000000
01111100
00000000
00000000
00000000
00000000
