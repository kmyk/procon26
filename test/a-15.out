-3 -1 H 0
2 3 H 270
2 6 H 0
7 3 H 0
7 11 H 90
12 15 H 90
13 13 H 90
20 13 H 0
20 20 H 90
